src/context/schemas/context.schema.json

```diff
@@ -4,11 +4,13 @@
   "type": "object",
   "properties": {
     "type": { "type": "string" },
-    "name": { "type": "string" },
+    "name": { "type": "string" }, 
     "id": {
+      "$comment": "Reserved for identifiers",
       "type": "object",
-      "additionalProperties": { "type": "string" }
-    }
+      "additionalProperties": { "type": "any" }
+    },
+    "additionalProperties": { "type": "any" }
```

Comment:
I think this is good to add, in addition should we add a `$comment` like above to indicate that other data goes here?